### PR TITLE
Test uses shared security group, no longer does restart test

### PR DESCRIPTION
**Pull request by Graeme-Miller**
CA uses subnet address

### CODE REVIEW

File: swarm/src/test/resources/docker/swarm.tests.bom [hunk 1]
```diff
@@ -396,6 +396,7 @@ brooklyn.catalog:
         swarm.port: 4000
         docker.recovery.stabilizationDelay: 10s
         docker.recovery.failOnRecurringFailuresInThisDuration: 5m
+        swarm.sharedsecuritygroup.create: true
```
Comment (drigodwin) [inline swarm/src/test/resources/docker/swarm.tests.bom]: This is going to be different on differing clouds which we test, do we want to put this here?

Comment (Graeme-Miller) [inline swarm/src/test/resources/docker/swarm.tests.bom]: I think so. swarm.sharedsecuritygroup.create is a brooklyn.parameter that is defaulted to 'true' in the main swarm blueprint. 

Setting it to true in the test ensure that we test what a user will get by default.

Comment (grkvlt) [inline swarm/src/test/resources/docker/swarm.tests.bom]: But the point is that on some clouds this default will not work, the user _must_ make a change from the default settings. Or are there going to be specific configurations for those clouds with this set to `false` as well?

Comment (grkvlt) [inline swarm/src/test/resources/docker/swarm.tests.bom]: @Graeme-Miller there should be tests with both _true_ and _false_ settings for this configuration key, on the appropriate clouds. But, to verify that the default setting work, we should *NOT* be setting it here, for the obvious reason that if the default was ever to change, we would not be "test[ing] what a user will get by default" any more.

File: swarm/src/main/resources/swarm/swarm.bom [hunk 2]
```diff
@@ -446,6 +446,10 @@ brooklyn.catalog:
           default: true
 
       brooklyn.config:
+        name: swarm.create.shared.securitygroup
```
Comment (grkvlt) [inline swarm/src/main/resources/swarm/swarm.bom]: I don't think this will work. It creates four config keys, `name`, `label`, `type`, and `default`.

**Pull request merged**